examples/weather_station/src/buzzer.cpp

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include "buzzer.h"
+
+#include <drivers/pwm.h>
+
+#define BUZZER_PWM_NODE DT_ALIAS(buzzer_pwm)
+#define BUZZER_PWM_NAME DT_LABEL(BUZZER_PWM_NODE)
+#define BUZZER_PWM_PIN DT_PROP(BUZZER_PWM_NODE, ch0_pin)
+#define BUZZER_PWM_FLAGS DT_PWMS_FLAGS(BUZZER_PWM_NODE)
+
+constexpr uint16_t kBuzzerPwmPeriodUs = 10000;
+constexpr uint16_t kBuzzerPwmDutyCycleUs = 5000;
+
+const device *kBuzzerDevice;
```

Comment:
It's not constant - don't use `k` prefix. Also, please put both in anonymous namespace or mark both as static constistently.